mews-operations/availabilityBlockProcessor.md

```diff
@@ -0,0 +1,176 @@
+# Channel Manager API Operations: Process availability blocks
+
+## Process availability block
```

Comment:
The name here is singular, but the URL name is plural. We should be consistent. I think singular is probably correct, because you can only process a single availability block with this operation.